crates/rspack_plugin_javascript/src/visitors/dependency/harmony_import_dependency_scanner.rs

```diff
@@ -16,17 +17,47 @@ use swc_core::{
   },
 };
 
-pub type ImportMap = FxHashMap<
-  (JsWord /* ident */, SyntaxContext /* ctxt */),
-  (JsWord /* request */, Option<JsWord> /* id */),
->;
-
 use crate::dependency::{
   HarmonyExportImportedSpecifierDependency, HarmonyImportDependency,
   HarmonyImportSpecifierDependency, Specifier,
 };
 
-pub type Imports = IndexMap<(JsWord, DependencyType), (Span, Vec<Specifier>, bool)>;
+pub struct ImporterReferenceInfo {
+  pub request: JsWord,
+  pub specifier: Specifier,
+  pub names: Option<JsWord>,
+}
+
+impl ImporterReferenceInfo {
+  pub fn new(request: JsWord, specifier: Specifier, names: Option<JsWord>) -> Self {
+    Self {
+      request,
+      specifier,
+      names,
+    }
+  }
+}
+
+pub type ImportMap =
+  FxHashMap<(JsWord /* ident */, SyntaxContext /* ctxt */), ImporterReferenceInfo>;
```

Comment:
IIRC, swc has a type alias represented this tuple type.

Reply:
https://github.com/web-infra-dev/rspack/blob/927278e367e87012018b9957ef2f233efd970367/crates/rspack_symbol/src/lib.rs#L4-L4